From cad731c0aa9cf7e07db0f61ba2abdf7583155883 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 23 Nov 2018 21:34:27 +0900
Subject: [PATCH 536/909] arm64: dts: r8a7795-salvator-xs: Enable LVDS output

commit 8f29eaaf48ebfaf1f170b7c569e5ca6014a4cdec from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch enable LVDS output for the Salvator-X 2nd version board
on the R8A7795 ES2.0 SoC.

[takeshi.kihara.df: rebased]
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts b/arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts
index fd45273343ba..6e4b068a6cfa 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-salvator-xs.dts
@@ -114,6 +114,16 @@
 	remote-endpoint = <&rcar_dw_hdmi1_out>;
 };
 
+&lvds0 {
+	ports {
+		port@1 {
+			lvds0_out: endpoint {
+				remote-endpoint = <&lvds_in>;
+			};
+		};
+	};
+};
+
 &ohci2 {
 	status = "okay";
 };
-- 
2.17.1

